sympy/functions/special/error_functions.py

```diff
@@ -2799,3 +2825,33 @@ def _eval_nseries(self, x, n, logx, cdir=0):
             f = self._eval_rewrite_as_intractable(*self.args)
             return f._eval_nseries(x, n, logx)
         return super()._eval_nseries(x, n, logx)
+
+
+class _expint(Function):
+    """
+    Helper function to make the $\\mathrm{expint}(nu, z)$
+    function tractable for the Gruntz algorithm.
+
+    """
+
+    def _eval_aseries(self, n, args0, x, logx):
+        from sympy.series.order import Order
+        point = args0[1]
+        nu = self.args[0]
+
+        if point is S.Infinity:
```

Comment:
We should have the following here !
```
if point in [S.Infinity, S.NegativeInfinity]:
```
On top of it, we could surely have checks for eg 
`expint(1, 1/x).as_leading_term(x, logx=None, cdir=1)` and `cdir = -1` returning `self`

Then if we add tests for limits eg limit(expint(1, 1/x), x, 0) (check both + and - ... we should get separate results) .... we then know for sure that the results are returned through gruntz as expected.